python/tvm/ir/op.py

```diff
@@ -115,3 +127,50 @@ def _register(v):
         return v
 
     return _register(value) if value is not None else _register
+
+
+def register_op_intrin_lowering(
+    op_name,
+    f=None,
+    target="default",
+    level=10,
+    override=False,
+):
```

Comment:
Let's avoid providing too many defaults :-)


```suggestion
def register_op_intrin_lowering(
    op_name,
    f,
    target,
    level=10,
    override=False,
):
```